### PR TITLE
Release notes for 2022-07-20

### PR DESCRIPTION
**Pull request by duskpoet**
This PR adds the Release Notes for the [2022-07-20 release of the Console](https://github.com/neondatabase/cloud/pull/1858/commits)

### CODE REVIEW

File: content/docs/release-notes.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 title: Release Notes
 ---
 
+## 2022-07-20 Console release
+
+### What's new
+
+* UI for pool enabling
```
Comment (stepashka) [inline content/docs/release-notes.md:9]:
```suggestion
* UI: add 'Enable pooling' toggle to the Project's Settings page 
```

**Review by stepashka — COMMENTED** [review]
Have we added more instances to the console as part of this release? This enables the 0 downtime deployment, i think we can mention that too.
We're also more precise when specifying the Postgres version, we can mention that too.
Even though this isn't published yet, we can mention that work has started on the api v2 and the spec is available now.

Anything else worth mentioning? Will add other reviewers here too.

**Comment by duskpoet** [comment]
> Have we added more instances to the console as part of this release? This enables the 0 downtime deployment, i think we can mention that too. We're also more precise when specifying the Postgres version, we can mention that too. Even though this isn't published yet, we can mention that work has started on the api v2 and the spec is available now.
> 
> Anything else worth mentioning? Will add other reviewers here too.

Unfortunately I am not really familiar with other changes. Maybe @ololobus @antonyc can help

**Review by ololobus — APPROVED** [review]
Pushed more details. I suggest to avoid mentioning v2 now

**Review by stepashka — APPROVED** [review]

**Pull request merged**